Examples/Algorithms/TrackFinding/src/TrackParamsLookupAccumulator.cpp

```diff
@@ -0,0 +1,73 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/TrackFinding/TrackParamsLookupAccumulator.hpp"
+
+#include "Acts/EventData/TrackParameters.hpp"
+#include "Acts/Utilities/Grid.hpp"
+#include "Acts/Utilities/GridIterator.hpp"
+
+void ActsExamples::TrackParamsLookupAccumulator::addTrack(
+    const Acts::CurvilinearTrackParameters& ipTrackParameters,
+    const Acts::CurvilinearTrackParameters& refTrackParameters,
```

Comment:
Separated the accumulation into the `Core` with the `Grid` type template. It will work with all the types of parameters. I'd prefer to keep `CurvilinearTrackParameters` in the examples to comply with the `PathSeeder` telescope seeding interface (plan to add examples for it a bit later)